consensus/istanbul/core/vrank_test.go

```diff
@@ -0,0 +1,128 @@
+package core
+
+import (
+	"encoding/hex"
+	"math/big"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/consensus/istanbul/validator"
+	"github.com/stretchr/testify/assert"
+)
+
+func genCommitteeFromAddrs(addrs []common.Address) istanbul.Validators {
+	committee := []istanbul.Validator{}
+	for _, addr := range addrs {
+		committee = append(committee, validator.New(addr))
+	}
+	return committee
+}
+
+func TestVrank(t *testing.T) {
+	var (
+		N         = 6
```

Comment:
Given 3f+2 total validators, the quorum must be 2f+2. With six validators (3f+2, where f=1), the quorum should be four to maintain system integrity even with a single faulty validator.

Reply:
I know but I wanted to test with a random quorum. If you think this is misleading, I'll change the quorum to four.

Reply:
fixed in 913085d9